src/models/image-map.ts

```diff
@@ -457,7 +457,7 @@ export const localAssetsImagesHandler: IImageHandler = {
     let _url = url;
     if (urlPieces && this.imageMap.unitCodeMap) {
       const urlUnitDir = urlPieces[1];
-      const newUnitDir = this.imageMap.unitCodeMap.get(urlUnitDir);
+      const newUnitDir = this.imageMap.unitCodeMap.get(urlUnitDir) || urlUnitDir;
```

Comment:
Without this change, URLs like `curriculum/sas/images/image.png` would be changed to `undefined/images/image.png` instead of `sas/images/image.png`.